cookbook/0005-image-service/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Recipe - Simple Manifest - Image with IIIF Image API Service
+layout: spec
+tags: [annex, service, services, specifications]
+cssversion: 2
+---
+
+This is a recipe from the [IIIF Cookbook][annex-cookbook].
+
+
+# Simple Manifest - Image with IIIF Image API Service
+
+## Use Case
+
+Paint a Canvas using an image with an associated IIIF Image API service.
+
+## Implementation notes
+
+The annotation structure follows that of the [Simplest Manifest - Image] recipe. Within the `body` of the image annotation, specify the IIIF Image API service using the `service` property. The `id` of the service is the base URI of the associated IIIF Image API service.  
+
+The `type` property is mandatory. Image API version 1 and 2 services may be referenced using the `type` values defined in the Presentation API[#service].  
+
+## Restrictions
+
+None known.
+
+## Example
+
+``` json-doc
+{
+  "@context": [
+    "http://www.w3.org/ns/anno.jsonld",
+    "http://iiif.io/api/presentation/{{ page.major }}/context.json"
+  ],
+  "id": "https://example.org/iiif/book5/manifest",
+  "type": "Manifest",
+  "label": { "en": [ "Simple Manifest - Image with IIIF Image API Service" ] },
+  "items": [
+    {
+      "id": "https://example.org/iiif/book5/canvas/p1",
+      "type": "Canvas",
+      "height": 1800,
+      "width": 1200,
```

Comment:
Should we add a label to the canvas (marked as recommended in the specification)

Reply:
Probably. I copied the example from mvm-image, which also lacks a label - but "minimum viable" is not equivalent to best practice.

Reply:
That makes sense 👍